<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCustomGalleryDialog</class>
 <widget class="QDialog" name="EditCustomGalleryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Custom Gallery</string>
  </property>
  <layout class="QVBoxLayout" name="plLayout">
   <item>
    <widget class="QLabel" name="plName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pleName"/>
   </item>
   <item>
    <widget class="QWidget" name="pwKeys" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="plKeys">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="plwContainsKeys"/>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="plEditKeys">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="ptbAddKey">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../GalleryExplorer.qrc">
             <normaloff>:/res/resources/left.png</normaloff>:/res/resources/left.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="ptbDelKey">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../GalleryExplorer.qrc">
             <normaloff>:/res/resources/right.png</normaloff>:/res/resources/right.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="plwKeys"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwButtons" native="true">
     <layout class="QHBoxLayout" name="plButtons">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="ppbSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pwBtnFree" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="ppbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GalleryExplorer.qrc"/>
 </resources>
 <connections/>
</ui>
